service-tests/github.js

```diff
@@ -205,3 +205,87 @@ t.create('hit counter for nonexistent repo')
     name: Joi.equal('goto counter'),
     value: Joi.string().regex(/^repo not found$/),
   }));
+
+t.create('open issues')
+  .get('/issues/badges/shields.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('issues'),
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]? open$/)
+  }));
+
+t.create('open issues (raw)')
+  .get('/issues-raw/badges/shields.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('issues'),
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]?$/)
+  }));
```

Comment:
What do you think about calling this `-minimalist` or `-clean`, or better yet, `/issues/minimalist`, rather than `-raw`? It seems to clarify that it's affecting the formatting, rather than the underlying number.

Reply:
I didn't introduce -raw, it was called like this before. I don't think we an break the existing interface.